.github/workflows/release.yml

```diff
@@ -23,6 +26,18 @@ jobs:
       with:
         generate_release_notes: true
         make_latest: true
+    - name: Log in to the Container registry
+      uses: docker/login-action@v3
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - uses: docker/setup-qemu-action@v3
+    - uses: docker/setup-buildx-action@v3
+    - uses: docker/build-push-action@v6
+      with:
```

Comment:
This needs to run after the `cargo publish` step. I think as written it might install the wrong version.

Reply:
I thought these were run sequentially.

Reply:
O yes. I think you're right, I misread this. Nevermind then.

Reply:
This could also be a separate job with a `needs: release` to ensure that it runs after the `release` job completes.